NeuroID/NeuroIDTracker.swift

```diff
@@ -10,25 +10,25 @@ import WebKit
 // MARK: - NeuroIDTracker
 
 public class NeuroIDTracker: NSObject {
-    private var screen: String?
+//    private var screen: String?
     private var className: String?
     private var createSessionEvent: NIDEvent?
     /// Capture letter count of textfield/textview to detect a paste action
     var textCapturing = [String: String]()
-    public init(screen: String, controller: UIViewController?) {
-        super.init()
-        self.screen = screen
-        if !NeuroID.isStopped() {
-            subscribe(inScreen: controller)
-        }
-        className = controller?.className
-    }
+//    public init(screen: String, controller: UIViewController?) {
+//        super.init()
+//        self.screen = screen
+//        if !NeuroID.isStopped() {
+//            subscribe(inScreen: controller)
+//        }
+//        className = controller?.className
+//    }
```

Comment:
Is there any reason not to remove this instead of commenting it out?

Reply:
Yes, this entire class will likely be removed, testing the interactions in our test apps. Few more updates before cleaning it all up.